opencti-platform/opencti-graphql/vitest.config.ci-unit.ts

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { buildTestConfig } from './vitest.config.test';
+
+export default buildTestConfig(['tests/(01|06)-*/**/*-test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']);
```

Comment:
```suggestion
export default buildTestConfig(['tests/(01|06|07)-*/**/*-test.{js,mjs,cjs,ts,mts,cts,jsx,tsx}']);
```
Sorry I changed a bit the numbers in on PR that I just merged.

Reply:
by the way 06 and 07 are not really unit, but I don't have a better idea on naming except "others"

Reply:
@aHenryJard Trying to add taxii test, I get an error. This one is not flaky.
Do you have an idea of the the origin ? 
If it's not obvious, would you agree if I dd this in the "out of scope" perimeter of this PR in order to unlock the merge of it ? 

https://github.com/OpenCTI-Platform/opencti/actions/runs/19576410474/job/56062919731
 ```
  × Should taxii collection coverage > should public taxii collection without user works 23ms
     → expected 0 to be greater than 0
   ✓ Should taxii collection coverage > should restricted taxii collection without user be forbidden 5ms

⎯⎯⎯⎯⎯⎯⎯ Failed Tests 1 ⎯⎯⎯⎯⎯⎯⎯

 FAIL  tests/06-httpTaxii/httpTaxii-test.ts > Should taxii collection coverage > should public taxii collection without user works
AssertionError: expected 0 to be greater than 0
 ❯ tests/06-httpTaxii/httpTaxii-test.ts:182:33
    180|     const content = await taxiiCollectionResponse.json() as { more: bo…
    181|     expect(content.more).toBeFalsy();
    182|     expect(content.next.length).toBeGreaterThan(0);
       |                                 ^
    183|   });
    184| 

```

Reply:
yes you can remove '07' for now, I will look at it

Reply:
@aHenryJard The issue was with "06" (taxii ) . I moved it to run it after integration test and it seems to work fine like this.